controllers/_application.js

```diff
@@ -257,28 +257,18 @@ module.exports = (app, passport, React, ReactDOMServer) => {
   // used for the mobile app
   app.post('/appLogin', (req, res, next) => {
     logger.info('\n\n\nappLogin\n\n\n');
-
-    logger.info(req.headers);
-
+    logger.info(JSON.stringify(req.headers));
     logger.info('\n\n\nappLogin\n\n\n');
+
     passport.authenticate('gravity-login', (err, userInfo) => {
       if (err) return next(err);
-      const accountData = JSON.parse(gravity.decrypt(userInfo.accountData));
 
-      // {
-      //   "account":"JUP-H496-3XEM-94ZH-3R548",
```

Comment:
Man I'm sure this was removed before... did we miss a merge?

Reply:
I have seen this before in the code but now that I can understand a little bit more the code I decided to remove it